wgsl/index.bs

```diff
@@ -3789,8 +3789,11 @@ When a variable is created, its memory contains an initial value as follows:
     * Otherwise, it is the result of evaluating the initializer expression at that point in the program execution.
 * For variables in the [=address spaces/workgroup=] address space:
     * When the store type is [=constructible=], the [=zero value=] for the store type.
-    * Otherwise, the store type is an array of construcible elements, and each element
-        is initialized to its zero value.
+    * If the store type is an [=atomic type=], the [=zero value=] is that of the underlying [=integer scalar=] type.
```

Comment:
Consider parenthesizing "([=integer scalar=])" to make it more clear that it's implied. (because atomics must be integer scalars (today))